src/routes/auth/+page.svelte

```diff
@@ -1,10 +1,17 @@
 <script lang="ts">
 	import { enhance } from '$app/forms';
 	import Button from '$lib/components/ui/Button.svelte';
+	import { onMount } from 'svelte';
 
 	const { form } = $props();
 
 	let formType = $state('login');
+	let redirect = $state('');
+
+	onMount(() => {
+		redirect = new URLSearchParams(location.search).get('redirectto') ?? '';
```

Comment:
can we just call it `redirect` instead of `redirectto`. If you want two words it should be `redirect-to` probably